pkg/services/apikey/apikeyimpl/store_test.go

```diff
@@ -53,9 +53,6 @@ func seedApiKeys(t *testing.T, store store, num int) {
 }
 
 func testIntegrationApiKeyDataAccess(t *testing.T, fn getStore) {
-	if testing.Short() {
```

Comment:
Seems like these tests are disabled. If they would be changed to `testIntegrationApiKeyDataAccess` you probably want to include the `if testing.Short() {` so might want to skip this change? If they should never run we should remove all the test I guess, but @grafana/grafana-authnz-team would have to answer based on they're responsible for the service account feature.

Reply:
They're actually not disabled, I just moved the check to the TestIntegration functions that call these to make things clear and easy to audit.  It is a bit confusing because of the tendency of some people to change the function name to disable it vs adding an explicit t.Skip call.  Not sure whether it makes sense to mark these explicitly as helpers or whether that would just be confusing when trying to debug a test failure.

Reply:
Ohh I see. Sorry for the noice